src/base/IoManip.h

```diff
@@ -76,20 +76,30 @@ operator <<(std::ostream &os, const RawPointerT<Pointer> &pd)
 }
 
 /// std::ostream manipulator to print integers as hex numbers prefixed by 0x
```

Comment:
The "prefixed by 0x" part of this documentation appears not to be true.

AFAICS most of the uses do actually need to display the value with that prefix. So perhapse it would be worth printing the prefix by default with a `noPrefix()` method to prevent its display. That would clean up a fair bit of mess in the debugs.

Reply:
That was my original implementation, using printPrefix(false) - positives are easier to understand.
Based on feedback from @rousskov , I’ve removed that code; for one, the cases where printing a prefix is NOT wanted are more than the opposite. The prefix can be printed by the surrounding code easily